generics/constants/endpoints.js

```diff
@@ -35,5 +35,7 @@ module.exports = {
     GET_FORM_DATA : "/plugin/v1/form/read",
     GET_SCHOOL_DATA : "/v1/org/search",
     USER_READ_V5 : "/v5/user/read",
-    USER_CONSENT_API: "/user/v1/consent/update"
+    USER_CONSENT_API: "/user/v1/consent/update",
+    GET_IMPORTED_SURVEY : "/v1/surveys/getStartedSurveys",
```

Comment:
@ankitpws update this as per latest API endpoint

Reply:
Updated